api/include/opentelemetry/trace/noop.h

```diff
@@ -102,15 +102,14 @@ class OPENTELEMETRY_EXPORT NoopTracer final : public Tracer,
 /**
  * No-op implementation of a TracerProvider.
  */
-class OPENTELEMETRY_EXPORT NoopTracerProvider final : public opentelemetry::trace::TracerProvider
+class OPENTELEMETRY_EXPORT NoopTracerProvider final : public trace_api::TracerProvider
```

Comment:
should we instead use `trace::TracerProvider`, and remove `trace_api` namespace alias from the header file. It's not recommended to have namespace alias in header files (though this is existing code)

Reply:
Yes, you are right. I'd change it as you wish.

Reply:
Looks like the `trace_api` alias defined here is been used in other places too. We can keep it here in that case.

Reply:
Reverted back only `trace_api` alias. However, I will take it as a note, maybe it would be removed after same changes are applied to SDK too.